04-cs-fundamentals/classroom/Interview-prorocess.md

```diff
@@ -0,0 +1,80 @@
+
+Before a phone screen it is a good idea to:
+
+- Review the job listing and take notes
+- Review your resume as the interviewer will likely ask questions drawn from your experience.
+- Prepare some questions to help you determine if the position is right for you.
+- Find a quiet place with good connectivity.
+
+### In-person Interview
+
+A technical in-person interview can vary greatly in length and process.  Typically however you will have some initial time to greet and introduce yourself to the interviewer(s) with some brief chitchat.  
+
+Then the interviewer will begin the technical portion of the interview.  This is often a _whiteboard interview_, meant to test your ability to solve new problems and communicate your thinking.  You will be given a problem and a whiteboard and asked to develop a solution and explain it to the interviewer.  
+
+
+## The Ada Internship-Interview Process
+
+As part of being an Ada student you will be placed in a internship with a sponsoring company.  To help determine placement and to give you practice interviewing, you will be assigned 6 Ada sponsors.  Over the course of a week, you will interview with each of the 6 companies/teams once in 30-minute technical interviews.  
+
+### Mock Interviews
+
+As an Ada student you will be matched up for a practice interview, called a "mock interview" with an experienced hiring manager or engineer.  They will give you a sample interview problem and conduct a practice interview with you.  After the "mock interview" the interviewer will write up a set of feedback for you to help you prepare for your actual internship interviews.
+
+## The Internship Interview
+
+Each internship interview is 30-minutes long and will consist of a few minutes of introduction, followed by a whiteboard question.  Make use of the [whiteboarding strategies](Whiteboarding-Tips.md#3-practice), we discuss in CS Fundamentals.  As a student, you are expected to prepare for each interview and try your upmost.  
+
+After the interviews conclude, both students & interviewers provide feedback to Ada staff which is used to decide placements.  Some feedback from interviewers will also be made available to students after internships begin.
+
+## Terminology
+
+- **Whiteboard Interview** - A Whiteboard interview is typically a problem solving exercise where a job applicant is asked to solve a programming or design problem on a whiteboard.  Applicants are assessed both on their _coding skills_ and ability to _communicate_ their reasoning.  Whiteboard interviews can, instead of using a physical whiteboard involve pair programming, or a shared screen environment.
+- **Phone Screen** - An initial telephone or internet chat with a job candidate to ensure the candidate's suitability for an in-peron interview.  This could involve a hiring manager, but is more likely to involve a recruiter.
```

Comment:
Consider moving phone screen up, since it happens earlier in the process and makes this flow a touch better.